src/test/scala/it/unibo/authsim/library/dsl/otp/generators/Helpers.scala

```diff
@@ -0,0 +1,24 @@
+package it.unibo.authsim.library.dsl.otp.generators
+
+import it.unibo.authsim.library.dsl.otp.model.OTP
+import it.unibo.authsim.library.dsl.policy.builders.StringPoliciesBuilders.OTPPolicyBuilder
+import it.unibo.authsim.library.dsl.policy.defaults.stringpolicy.OTPPolicyDefault
+import it.unibo.authsim.library.dsl.policy.model.StringPolicies.OTPPolicy
+
+import scala.language.postfixOps
+
+object Helpers:
```

Comment:
This object (and the file) could be renamed to specify that it is used in tests, but it is not that important.